cmd/bucket-replication.go

```diff
@@ -2860,8 +2863,13 @@ func (p *ReplicationPool) queueMRFSave(entry MRFReplicateEntry) {
 	select {
 	case <-GlobalContext.Done():
 		return
-	case p.mrfSaveCh <- entry:
+	case <-p.mrfStopCh:
```

Comment:
We could bring the inner select out, it has the same effect.

Reply:
bringing out the inner select can cause panic due to sending msg to closed channel. select doesn't have ordering - there are m senders and 1 receiver here.